### ISSUE TITLE
Tile 8264 on 2024-10-07 has a spike of ELG targets at z=6

### ISSUE THREAD

**Issue by dstndstn** [desihub/desisurveyops#228]
Exp 256817

![1](https://github.com/user-attachments/assets/516dd3c3-5a79-426a-9f43-fa781c75fa33)

**Comment by dstndstn** [comment]
In the sframesky plot, there is large broad-band excess light in all the z spectra (very different than neighbouring exposures).

![2](https://github.com/user-attachments/assets/2bb211a8-ddca-4f19-b652-2c45ff66683e)

**Comment by schlafly** [comment]
Weird.  I can't recall seeing one like this before.  I took a look at Anand's space watch movie and didn't see anything.  We should discuss, but likely it ends up getting marked bad and reprocessed.  Next steps to figure out what went wrong would be to look at the spectra before sky subtraction to see if anything went wrong there and to plot the sky brightness on each fiber to see if there is a clear trend on the CCD or on the sky.

**Comment by schlafly** [comment]
But we should definitely mark unsure for now!

**Comment by araichoor** [comment]
sorry in advance if that is irrelevant, but just in case:

that exposure 256817 started at 00:45 (the next one started at 01:01).

just browsing the https://telemetry.desi.lbl.gov/ page looking for something in that time range, and I see a change in the lower shutter (between 00:46 and 00:48, the only one during the night):
<img width="1354" alt="Screenshot 2024-10-10 at 3 48 36 PM" src="https://github.com/user-attachments/assets/fb6796a3-db06-4011-8730-a5d37d9418e2">
could that be related? (e.g. if that "lower" is the one in front of the z-cameras).

**Comment by araichoor** [comment]
if useful, here are a couple of plots for  520 `OBJTYPE="SKY"` fibers in the sframe files, focusing on the 8000A-9000A range.
from those it looks to me that the issue is with the `FIBERASSIGN_Y<200` part the focal plane... I don t know if there s a story where that would make sense.

here are the sky spectra, with a per-petal offset for clarity:
![sframe-sky-00256817-spectra](https://github.com/user-attachments/assets/56374efe-b60d-4575-9d8c-9607db69f37f)
petals 5 and 6 look not really affected, and petal 4 slightly affected.

and here s when I look at this per-fiber quantity `SFRAME_FLUX_8250_8700`, which is the ivar-weighted mean of the spectra in the 8250A-8700A region:
- left plots: as a function of `FIBER` or `FIBERASSIGN_Y`;
- right plot: highlighting in the focal plane the fibers with large values.
![sframe-sky-00256817-flux](https://github.com/user-attachments/assets/3cbee715-6662-4599-a361-2fde37af767c)

happy to look at any further suggestions!

**Comment by schlafly** [comment]
Maybe just restating your figures---the statement is that the ~bottom half of the focal plane saw a lot of light from 8300-8700 A and no where else.  The light came from outside the system since it's coherent across the focal plane and not from spectrograph to spectrograph.  The spectrum is a broad bump.  

It might be interesting to learn more to figure out where the light is coming from (someone shining something at the dome, walking around the dome, ...?), but it's super rare so it's unclear to me we care.

I don't think this could "really" be part of the astronomical sky.  I vote we get the data team to mark this as bad and reprocess.

**Comment by sbailey** [comment]
FWIW, this exposure was not adjacent to the visitor center caravan which left at 21:06 that night, hours before this exposure at 00:45.  But the symptoms are what I would suspect from stray light from a car headlight or flashlight.  I agree with marking this as bad and reprocessing.  @abhi0395 @akremin

**Comment by akremin** [comment]
I've marked exposure 256817, night 20241007, tile 8264 as bad and purged the processed data. That was the only observation of the tile, so I reran the tsnr and nightqa to reflect that there are no good observations.

Back to surveyops to update the specstatus file to have 0s EFFTIME and reset the QA.

**Comment by geordie666** [comment]
Thanks, Anthony. I updated the specstatus file and I'm closing this one.

**Comment by djschlegel** [comment]
I noticed this tile is still blocking observations, and is marked QA='none' in the tiles-specstatus.ecsv file.  Am I mis-interpreting that?

**Comment by geordie666** [comment]
We zeroed out the bad exposure of this tile from night `20241007`. This would have set QA='none' (correctly) and reset the `EFFTIME` to zero. So, then the overall behavior would proceed as if this tile had no good observations.

We then re-observed this tile on night `20241024` and it achieved sufficient `EFFTIME`. So, this tile should now be done. It should therefore pop up in the regular tile QA as part of night `20241024`, and, indeed, I see it here:

https://data.desi.lbl.gov/desi/spectro/redux/daily/tiles/cumulative/8264/20241024/tile-qa-8264-thru20241024.png

Bottom line: I think this tile should have been picked up as part of the regular daily tile QA. @djschlegel: Did you not see tile `8264` when you did tile QA for `20241024`?

**Comment by djschlegel** [comment]
OK, great.  I've not yet done QA for 20241024, which I'll do today.
